### PR TITLE
feat(doc): add readme

### PR DESCRIPTION
**Pull request by uguy**
- added README.adoc
- moved arg validation to yargs check()

### CODE REVIEW

File: cli/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "clean": "rimraf build .cache"
   },
-  "devDependencies": {
```
Comment (benjaminParisel) [inline cli/package.json:12]: Why did you move this devDependencies (only use on dev time i guess), to dependencies ?

**Review by benjaminParisel — APPROVED** [review]

File: cli/package.json (hunk 1 above)
Comment (uguy) [reply cli/package.json:12]: Because they are actually not dev deps be deps for the CLI to work if we were to release it.
